apps/docs/src/pages/en/guides/pipelines.mdx

```diff
@@ -21,12 +22,14 @@ are what you will be attaching pipelines to.
 
 ## What are Pipelines?
 
-Pipelines are logical checks and rules data must pass through and clear so it 
-can continue to be processed by your applications, and subsequently sent 
-"downstream" (i.e. written to a database, or passed to another application or 
-service). Pipelines are put together in the form of steps (which we refer 
-to interchangeably as rules), and each step has a type which determine the 
-kinds of actions to take on data.
+In simple words, it's a workflow for data.
```

Comment:
"Pipelines in Streamdal work similarly to pipelines in other products that utilize step-based workflows."

Reply:
Updated